.github/workflows/drizzle_ci.yml

```diff
@@ -104,3 +107,21 @@ jobs:
       uses: codecov/codecov-action@v1
       with:
         file: ./coverage.xml
+
+  bandit:
+    name: Bandit
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install bandit
+    - name: Security checks with bandit
+      run: |
+        bandit romancal -r -x tests
```

Comment:
This is not doing anything I suspect, as there's no `romancal` in `drizzle`.

Reply:
The github actions workflow has been modified to use `tox` now and this mistake has been corrected.